### PR TITLE
Document using volume mounts with Docker image

### PR DESCRIPTION
**Pull request by pierwill**
Work toward #1947.

### CODE REVIEW

File: content/influxdb/v2.0/get-started.md [hunk 1]
```diff
@@ -330,14 +330,31 @@ docker run -p 8086:8086 quay.io/influxdb/influxdb:v2.0.4 --reporting-disabled
...
-### Console into the InfluxDB Container (Optional)
+### Console into the InfluxDB container (optional)
 
 To use the `influx` command line interface, console into the `influxdb` Docker container:
 
 ```bash
 docker exec -it influxdb /bin/bash
 ```
 
+### Persist data outside the InfluxDB container (optional)
```
Comment (sanderson) [inline content/influxdb/v2.0/get-started.md:341]: I don’t know that’d I’d even say this is optional. Mounting a local directory should be the default behavior we suggest. If they don’t specify a local mount, they will lose all their data any time the container restarts.

File: content/influxdb/v2.0/get-started.md [hunk 2]
```diff
@@ -315,6 +315,32 @@ docker run --name influxdb -p 8086:8086 quay.io/influxdb/influxdb:v2.0.4
 ```
 _To run InfluxDB in [detached mode](https://docs.docker.com/engine/reference/run/#detached-vs-foreground), include the `-d` flag in the `docker run` command._
 
+### Persist data outside the InfluxDB container
+
+1. Create a new directory in which to store your data, and change to that directory.
+   ```sh
+   mkdir my-influxdb-docker-data
+   cd my-influxdb-docker-data/
+   ```
```
Comment (sanderson) [inline content/influxdb/v2.0/get-started.md:324]:
```suggestion
1. Create a new directory to store your data in and navigate into the directory.
   
   ```sh
   mkdir path/to/influxdb-docker-volume && cd $_
   ```
```

File: content/influxdb/v2.0/get-started.md [hunk 3]
```diff
@@ -315,6 +315,32 @@ docker run --name influxdb -p 8086:8086 quay.io/influxdb/influxdb:v2.0.4
...
+2. From within your new data directory,
+   run the InfluxDB Docker container with the `-v` flag as in the example below.
+   This will persist the data from `/root/.influxdb2/` _inside_ the container to `my-influxdb-docker-data/` on the host file system.
+   ```sh
+   docker run \
+       --name influxdb \
+       -p 8086:8086 \
+       -v $PWD:/root/.influxdbv2 \
+       quay.io/influxdb/influxdb:v2.0.4 && \
+   ```
```
Comment (sanderson) [inline content/influxdb/v2.0/get-started.md:334]:
~~~suggestion
2. From within your new directory, run the InfluxDB Docker container with the `--volume` flag to
   persist data from `/root/.influxdb2/` _inside_ the container to the current working directory in
   the host file system.
   
   ```sh
   docker run \
       --name influxdb \
       -p 8086:8086 \
       --volume $PWD:/root/.influxdbv2 \
       quay.io/influxdb/influxdb:v2.0.4
   ```
~~~

**Review by sanderson — COMMENTED** [review]
Some suggestions

File: content/influxdb/v2.0/get-started.md (hunk 2 above)
Comment (pierwill) [reply content/influxdb/v2.0/get-started.md:324]: `path/to/influxdb-docker-volume` sounds to me like something that should already exist... How about `path/to/influxdb-docker-data-volume`?

**Review by sanderson — APPROVED** [review]

**Pull request merged**